use-submit-node.md

```diff
@@ -34,16 +34,16 @@ guide](connecting).
 Data Practices
 --------------
 
-**We do not back up data on CHTC submission points**, and users should
+**CHTC data locations are not backed up**, and users should
```

Comment:
are we okay with saying "file" locations instead? Not all stuff on CHTC is data. 
That's being pedantic, so I'm okay being overruled if you think "data" is easier to understand.